### PR TITLE
Adding a page for the virtual notebook

### PR DESCRIPTION
**Pull request by mtjandra**
* Added a blank page with a header for the virtual notebook.
* Can navigate from hamburger menu.
* Loads notes

demo

### CODE REVIEW

**Comment by lawong-d2l** [comment]
Do we need to add this page to the `fragments` array in the polymer.json file?

File: src/my-virtual-notebook.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import {PolymerElement} from '@polymer/polymer/polymer-element.js';
+import {PrefetchMixin} from './prefetch-mixin.js';
+import {SirenEntityMixin} from './siren-entity-mixin.js';
+import {html} from '@polymer/polymer/lib/utils/html-tag.js';
+
```
Comment (lawong-d2l) [inline src/my-virtual-notebook.js]: I think we still need these comments in there. Not sure if it's unnecessary with Polymer 3, but it's bit me in the past before with other projects. Especially with linting I think.

/* @mixes PrefetchMixin
   @mixes SirenEntityMixin */

**Comment by mtjandra** [comment]
> Do we need to add this page to the fragments array in the polymer.json file?

I don't think so. Right now virtual notebook is imported by the app shell, and doesn't seem to be loaded lazily, if I put a logging statement I see it appear on the login page. But I may not be interpreting the [docs](https://www.polymer-project.org/3.0/docs/tools/polymer-json#fragments) right. Let me know if I'm missing something.

File: src/my-virtual-notebook.js (hunk 1 above)
Comment (mtjandra) [reply src/my-virtual-notebook.js]: I'm able to run `npm run lint` successfully without the comments. Let me know if there is another linting stage.

File: src/my-virtual-notebook.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+			'_changed(entity)'
+		];
+	}
+
+	static get is() {
+		return 'my-virtual-notebook';
+	}
+
+	_changed() {
+		this._getUrlForNotes(this.href);
```
Comment (mtjandra) [inline src/my-virtual-notebook.js]: Going to play with this a little more. I was expecting to be passed down the json object from the specified href, but it was not what I received.

**Pull request merged**